lib/ansible/modules/network/nxos/nxos_vtp_password.py

```diff
@@ -218,12 +218,6 @@ def main():
         if vtp_password is not None:
             if existing['vtp_password'] == proposed['vtp_password']:
```

Comment:
Idempotence over removal is probably better served with

```
if not existing['vtp_password']:
    pass
```

above this line, and this line becoming an `elif`. This fails to error on idempotent check, but _does_ error on the case of mismatched passwords, which will otherwise be silently ignored, which does not seem to be desired here.